src/test/infrastructure/nfs-server/values.yaml

```diff
@@ -0,0 +1,75 @@
+# Default values for nfs-server.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: docker.atl-paas.net/atlassian/nfs-server-test
+  #pullPolicy: IfNotPresent
+  pullPolicy: Always
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "wkritzinger"
+
+imagePullSecrets:
+  - name: regcred
+nameOverride: ""
```

Comment:
I believe you can leave out `""` for empty values.

Reply:
This came from the helm chart template, as did most of these values in this file so I might leave it as-is